### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by Arshad-Saira**
This is test code for PR practice

### CODE REVIEW

File: code/p1.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("this is in test code branch")
```
Comment (mhassan-mustafa) [inline code/p1.py]: First latter "t" must be capital...

Comment (Iihtisham12) [inline code/p1.py]: I Write Some Code For More Practice

Comment (mhassan-mustafa) [inline code/p1.py]: `print("Hello World...")`

Comment (Iihtisham12) [inline code/p1.py]: def sum(a,b):
    c = a + b
    retrun c
sum(12,44)

File: code/p1.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
```
Comment (mhassan-mustafa) [inline code/p1.py:2]: First latter "t" must be capital...

**Review by mhassan-mustafa — COMMENTED** [review]
hello...

**Comment by Arshad-Saira** [comment]
> hello...

hello

File: code/p1.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
```
Comment (Iihtisham12) [inline code/p1.py:3]: Remove Three Line

File: code/p1.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+print("this is in test code branch")
```
Comment (Iihtisham12) [inline code/p1.py:1]: uyfuy

File: code/p1.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+
+def test_sum(sum):
+    if (1+5) == sum:
+        print("True")
+    else:
+        print("False")
+        
+test_sum(6)
```
Comment (mhassan-mustafa) [inline code/p1.py]: update it as follows:
`test_sum(10)`

File: code/p1.py (hunk 3 above)
Comment (Iihtisham12) [inline code/p1.py:3]: Add Here Some Coments

File: code/p1.py (hunk 5 above)
Comment (Arshad-Saira) [reply code/p1.py]: Done

**Review by mhassan-mustafa — APPROVED** [review]

**Review by Iihtisham12 — APPROVED** [review]

**Comment by Arshad-Saira** [comment]
Done

**Pull request merged**